tests/subsys/fw_info/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ cmake_minimum_required(VERSION 3.8.2)
 # boilerplate.cmake.
 set(provision_hex provision.hex)
 #set(provision_hex provision_nrf9160.hex)
+#set(provision_hex provision_nrf5340_cpuapp.hex)
```

Comment:
Should we consider writing separate test projects for these boards? We want to avoid having these commented out variants at least, and make it more visible that these variants can be tested. Maybe this is something we should create a task for and fix later.

Reply:
Changed it so you pass provision_hex to cmake.

    cmake ... -Dprovision_hex=provision_nrf5340_cpuapp.hex